app/models/pg_rls/admin.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module PgRls
+  # The admin class provides the admin_execute method to execute
+  # queries on the admin shard.
+  class Admin
+    def self.admin_execute(sql = nil)
+      PgRls::Record.connected_to(shard: :admin) do
+        return yield.presence if block_given?
+
+        PgRls::Record.connection.execute(sql).presence
+      end
+    end
```

Comment:
redundant naming, and we must add an alias to the module it self with the deprecation warning for the next mayor upgrade, same way as we did with the other method